tribits/ReleaseNotes.txt

```diff
@@ -2,7 +2,14 @@
 Release Notes for TriBITS
 ----------------------------------------
 
-2108/10/10:
+2020/02/07:
+
+(*) MAJOR: TriBITS Core: CMAKE_CXX_STANDARD is now always set to at least 11
```

Comment:
We need to also add that the `-std=c++11` option will be removed from `CMAKE_CXX_FLAGS` from the generated and installed `<Project>Config.cmake`, `<Package>Config.cmake`, `Makefile.export.<Project>`, and `Makefile.export.<Package>` files.  This does not really break CMake clients that pull in `<Project>Config.cmake` or `<Package>Config.cmake` because they will get what they need through the imported targets but it will break Makefile clients that expect to get `-std=c++11` through the `CMAKE_CXX_FLAGS` through the imported `Makefile.export.<Project>` or and `Makefile.export.<Package>` files.  But this does not need much of an explanation given the desire to drop support for these files as per #299.

Reply:
Thanks.  I've updated the release notes for this.